tests/bootstrap.php

```diff
@@ -13,7 +13,7 @@
 }
 
 // Give access to tests_add_filter() function.
-require_once 'includes/functions.php';
+require_once $_tests_dir . '/includes/functions.php';
```

Comment:
Maybe this isn't needed for this repo, but this prevented a fatal error in my environment.

Reply:
np. 👍